### PR TITLE
adding PPE redirect to main

### PR DESCRIPTION
**Pull request by DarrelDonald**

### CODE REVIEW

**Review by rekhaswaminathan — APPROVED** [review]

File: AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.html [hunk 1]
```diff
@@ -228,3 +228,11 @@ <h4 class="modal-title">Access Denied</h4>
     </split>
   </div>
 </div>
+<div *ngIf="!isPPE && detectorGraduation" style="display: flex;">
```
Comment (ShekharGupta1988) [inline AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.html:231]: will this work locally? 
Would it be better to have a check like (i.e. check if the env is prod)
<div *ngIf="isProd && detectorGraduation">...

File: AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.html [hunk 2]
```diff
@@ -228,3 +228,11 @@ <h4 class="modal-title">Access Denied</h4>
...
+  <status-icon [status]="HealthStatus.Warning" class="mr-3"></status-icon>
+  <div>
+    <p>To ensure the quality, accuracy, and security of detectors, the <b>{{resourceService["searchSuffix"]}}</b> team is now utilizing the <a [href]="PPELink">AppLens PPE Environment</a> for detector development.</p>
+    <p>To learn more about this new experience, click <a href="https://applensdiagnostics.azurewebsites.net/?p=715">here</a>.</p>
```
Comment (ShekharGupta1988) [inline AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.html:235]: in <a tag, have target='_blank' to open the doc link in a new tab.

File: AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.ts [hunk 3]
```diff
@@ -161,6 +167,25 @@ export class OnboardingFlowComponent implements OnInit {
       this.initialized = true;
       this._telemetryService.logPageView(TelemetryEventNames.OnboardingFlowLoaded, {});
     }
+
+    this.detectorGraduation = true;
+    this.diagnosticApiService.getDevopsConfig(`${this.resourceService.ArmResource.provider}/${this.resourceService.ArmResource.resourceTypeName}`).subscribe(devopsConfig => {
+      this.detectorGraduation = devopsConfig.graduationEnabled;
+
+      this.diagnosticApiService.getDetectorDevelopmentEnv().subscribe(env => {
+        this.PPELink = `https://applens-ppe.trafficmanager.net/${this._router.url}`
```
Comment (ShekharGupta1988) [inline AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.ts:176]: better to move the hostname of PPE env as a app setting. (For soverign clouds support in future)

File: AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.ts [hunk 4]
```diff
@@ -161,6 +167,25 @@ export class OnboardingFlowComponent implements OnInit {
...
+        this.isPPE = env === "PPE";
```
Comment (ShekharGupta1988) [inline AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.ts:177]: as commented above, better to check for isProd = env === 'PROD';

**Review by ShekharGupta1988 — APPROVED** [review]

File: AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.html (hunk 2 above)
Comment (ShekharGupta1988) [reply AngularApp/projects/applens/src/app/modules/dashboard/onboarding-flow/onboarding-flow.component.html:235]: @DarrelDonald - This is not done yet?

**Review by ShekharGupta1988 — APPROVED** [review]

**Pull request merged**